app/tests/test_models.py

```diff
@@ -1,7 +1,39 @@
+from flask import Flask
+from flask.ext.testing import TestCase
 from nose.tools import eq_
 
 from .. import models
 
+db = models.db
+
+class DbTestCase(TestCase):
+    BASE_APP_CONFIG = dict(
+        # TODO: We should use a test postgres db instead, since it's
+        # closer to our production deployment.
```

Comment:
I'm not sure what the best way to create a test posgres DB is; should we do it in Python as part of the whole test suite setup, or should we do it as part of a Dockerfile, or something else?